.pipelines/shared-cluster.yml

```diff
@@ -0,0 +1,97 @@
+# No PR triggers as it runs on a schedule
+pr: none
+trigger: none
+
+schedules:
+- cron: "0 1 * * Sun"
+  displayName: ⭕ Rebuild shared cluster Sun 1am (UTC)
+  always: true
+  branches:
+    include:
+    - master
+
```

Comment:
up to now we've been coding this manually in vsts - see the mirror pipelines as an example.  If doing it this way works, probably worth moving the mirror pipelines to match this.

Reply:
I sure hope so, that's what the docs say https://docs.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers?view=azure-devops&tabs=yaml